src/core/scopes.cc

```diff
@@ -1086,6 +1087,32 @@ int32_t scopes::load_parts(waflz_pb::scope& a_scope,
         return WAFLZ_STATUS_OK;
 }
 //: ----------------------------------------------------------------------------
+//: \details extern function to call process and pass on event info
+//: \return  TODO
+//: \param   TODO
+//: ----------------------------------------------------------------------------
+int32_t scopes::process_request_plugin(char **ao_event,
+                                       void *a_ctx,
+                                       const rqst_ctx_callbacks *a_callbacks,
+                                       rqst_ctx **ao_rqst_ctx)
+{
+        waflz_pb::event *l_audit_event = NULL;
+        waflz_pb::event *l_prod_event = NULL;
+        const waflz_pb::enforcement *l_enf = NULL;
+        int32_t l_s;
+        l_s = process(&l_enf, &l_audit_event, &l_prod_event, a_ctx, PART_MK_ALL, a_callbacks, ao_rqst_ctx);
+        // TODO: dev handle both audit and prod
```

Comment:
is this TODO for later?

Reply:
i mean for next PR?

Reply:
Yeah, lot of TODOs, for now just in a POC mode to send an alert. Will follow up with more changes to this func